### PR TITLE
Fix SQL typo

### PR DESCRIPTION
**Pull request by alphaX86**
I've found typo in your README file and got it covered! Let me know if I need to make any changes! :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Appsmith doesn't take the fun out of coding, because it treats every block as an
 * **JS Logic**: Write snippets of business logic using JS to transform data, manipuate UI or trigger workflows. Use popular libraries like lodash & moment anywhere in the app
 * **Data Workflows**: Simple configuration to create flows when users interact with the UI.
 * **Realtime Editor**: Changes in your application reflect instantly with every edit. No need to compile!
-* **Works with existing, live databases**: Connect directly to any Postgres, MySQL, & Mongo db
+* **Works with existing, live databases**: Connect directly to any PostgreSQL, MySQL, MongoDB and with any SQL Databases
```
Comment (alphaX86) [inline README.md:62]: Changes made...

**Review by Nikhil-Nandagopal — CHANGES_REQUESTED** [review]
Hi, we don't support all SQL databases. Please keep the sentence the same with only the typo changes 🙂

**Comment by alphaX86** [comment]
Got it! Wait a sec....

File: README.md [hunk 2]
```diff
@@ -55,11 +55,11 @@ Appsmith doesn't take the fun out of coding, because it treats every block as an
 
 * **5 minute setup**: Deploy Appsmith on your servers in 5 minutes.
 * **Build custom UI**: Drag & drop, resize and style widgets **without HTML / CSS**.
-* **Query data**: Query & update your database directly from the UI. Connect to **postgres, mongo, MySQL, REST & GraphQL APIs**.
+* **Query data**: Query & update your database directly from the UI. Connect to **PostgreSQL, MongoDB, MySQL, REST & GraphQL APIs**.
```
Comment (alphaX86) [inline README.md:58]: Typos fixed

File: README.md [hunk 3]
```diff
@@ -55,11 +55,11 @@ Appsmith doesn't take the fun out of coding, because it treats every block as an
...
 * **JS Logic**: Write snippets of business logic using JS to transform data, manipuate UI or trigger workflows. Use popular libraries like lodash & moment anywhere in the app
 * **Data Workflows**: Simple configuration to create flows when users interact with the UI.
 * **Realtime Editor**: Changes in your application reflect instantly with every edit. No need to compile!
-* **Works with existing, live databases**: Connect directly to any Postgres, MySQL, & Mongo db
+* **Works with existing, live databases**: Connect directly to any PostgreSQL, MySQL & MongoDB
```
Comment (alphaX86) [inline README.md:62]: As you've requested! :)

**Review by Nikhil-Nandagopal — APPROVED** [review]

**Pull request merged**